### PR TITLE
984 Add Google Tag Manager

### PR DESCRIPTION
**Pull request by kenjinp**
### How to test

- Start Nexus Web app with GTM_CODE='Your gmt code'

- Open Nexus Web and notice the following popup:

Screenshot 2020-01-24 at 16 12 14

- Click 'Allow' - gtm tag should appear in the page's html. You can also check that preferences are saved to the localStorage.

Screenshot 2020-01-24 at 14 02 04

- To delete gtm, click the 'info' icon on the Header. There you will see the button 'Remove tracking preferences'. Click it, and It will reset your settings in the localStorage, remove the gtm script from the page, and launch same Popup again. If you close it or click 'Don't allow', it will save your preferences to thelocalStorage.

Screenshot 2020-01-24 at 13 47 30

### CODE REVIEW

File: src/shared/layouts/MainLayout.tsx [hunk 1]
```diff
@@ -55,6 +58,8 @@ const MainLayout: React.FunctionComponent<MainLayoutProps> = ({
     })
   );
 
+  console.log('gtmCode', gtmCode);
```
Comment (dhaneshnm) [inline src/shared/layouts/MainLayout.tsx]: Do we need to log the code in to console?

File: src/shared/containers/ConsentContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        consentToTracking: allow,
+        hasSetPreferences: true,
+      });
+    });
+  }
+
+  if (consent && consent.consentToTracking) {
+    // do google stuff
+    enableTracking(trackingCode);
+    console.log('allowing tracking.....');
```
Comment (dhaneshnm) [inline src/shared/containers/ConsentContainer.tsx]: Do we need to log this?

File: src/shared/layouts/MainLayout.tsx (hunk 1 above)
Comment (Stafeeva) [reply src/shared/layouts/MainLayout.tsx]: It is a draft PR, will be removed

File: src/shared/containers/ConsentContainer.tsx (hunk 2 above)
Comment (Stafeeva) [reply src/shared/containers/ConsentContainer.tsx]: It is a draft PR, will be removed

**Pull request merged**